.jshintrc

```diff
@@ -7,7 +7,7 @@
   "forin": true,
   "freeze": true,
   "indent": 2,
-  "latedef": true,
+  "latedef": false,
```

Comment:
I'm not sure exactly what happened, but some new version of some npm module is causing a bunch of "used before defined" errors when linting. I don't have a ton of time to investigate and couldn't figure it out right away, so I'm just temporarily turning it off :/